pkg/sigma/v2/tree.go

```diff
@@ -204,6 +204,7 @@ func extractAndBuildBranches(d Detection, g string) ([]Branch, error) {
 func extractWildcardIdents(d Detection, g string) ([]interface{}, error) {
 	rules := make([]interface{}, 0)
 	for k, v := range d {
+		//I don't think this needs to use the other glob library?
```

Comment:
Maybe not, but I would prefer to stick to just one library for keeping external imports to a minimum. 

Plus, I think even if new code is more complex, at least it's consistent across the codebase.

Reply:
Changed with with 4e34d32336a852448f9ef77829afde342bc1416b, v2 now uses globwas everywhere.